src/test/kotlin/com/hibob/academy/dao/PetDaoTest.kt

```diff
@@ -0,0 +1,104 @@
+package com.hibob.academy.dao
+
+import PetDao
+import com.hibob.academy.utils.BobDbTest
+
+import org.jooq.DSLContext
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.sql.Date
+import java.util.*
+import kotlin.random.Random
+
+
+@BobDbTest
+class PetDaoTest @Autowired constructor(private val sql: DSLContext){
+
+    private val petDao = PetDao(sql)
+    private val table = PetTable.instance
+    val companyId = Random.nextLong()
+    val id = 1L
+    val ownerId =  0L  // no owner
```

Comment:
Why not null? 
OwnerId should be nullable, if no owner fir this pet then no id should be provided.